libcontainer/process.go

```diff
@@ -47,6 +47,10 @@ type Process struct {
 	// ExtraFiles specifies additional open files to be inherited by the container
 	ExtraFiles []*os.File
 
+	// Initial sizings for the console
+	ConsoleWidth  uint
+	ConsoleHeight uint
```

Comment:
Since we have to use `uint16` in the `Resize` why not just make these `uint16` in the first place?